src/test/util/file_util/add_and_remove_slash_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group